### ISSUE TITLE
Switch from gocui to tview

### ISSUE THREAD

**Issue by docwhat** [wagoodman/dive#261]
The [gocui](https://github.com/jroimartin/gocui) seems to be dead (no activity for 2 years) and is built on [termbox-go](https://github.com/nsf/termbox-go) which is unsupported now.

The new hotness seems to be [tview](https://github.com/rivo/tview) which is built on [tcell](https://github.com/gdamore/tcell) (I guess?)

Either way, I see problems with terminals, etc. and it seems like using a better terminal library would help.

**Comment by wagoodman** [comment]
I agree but haven't had any pressing issues that have forced me to refector just yet. I have been tracking the effort of @mjarkk and others on https://github.com/awesome-gocui/gocui/pull/45 , an implementation of gocui on tcell. This would make the migration effort pretty simple when/if completed.
